### ISSUE TITLE
process exited with code 3228369022

### ISSUE THREAD

**Issue by Jeffreywindows** [otya128/winevdm#1412]
Hello. I seem to encounter this issue and immediately closes.
I installed c++ visual studio 2017 and seemed to be unable to run old applications.
The apps I tried to run are Exile 1 and 2 and Bad toys 3D

**Comment by cracyc** [comment]
Maybe try the gcc built version you can get from https://ci.appveyor.com/project/otya128/winevdm/builds/49795086/job/uqh0ha83siyeyja0/artifacts

**Comment by Jeffreywindows** [comment]
> Maybe try the gcc built version you can get from https://ci.appveyor.com/project/otya128/winevdm/builds/49795086/job/uqh0ha83siyeyja0/artifacts

Hello there and thanks for the quick response! Just before you replied I went and tested the latest unstable version and that seemed to have worked and was able to play the game. Is there a possibility that version 0.9 doesn't work on windows 11? I saw YouTube videos and those content creators used an older version and worked for them.

Shall I close this thread or keep it open as a possible bug report?
